xds/src/test/java/io/grpc/xds/XdsSdsClientServerTest.java

```diff
@@ -285,27 +292,21 @@ public void mtlsClientServer_changeServerContext_expectException()
     }
   }
 
-  private XdsClient.LdsResourceWatcher performMtlsTestAndGetListenerWatcher(
+  private void performMtlsTestAndGetListenerWatcher(
       UpstreamTlsContext upstreamTlsContext, boolean newApi, String certInstanceName2,
       String privateKey2, String cert2, String trustCa2)
-      throws IOException, URISyntaxException {
+      throws Exception {
     DownstreamTlsContext downstreamTlsContext =
         setBootstrapInfoAndBuildDownstreamTlsContext(certInstanceName2, privateKey2, cert2,
             trustCa2, true, true);
 
-    final XdsClientWrapperForServerSds xdsClientWrapperForServerSds =
-        createXdsClientWrapperForServerSds(port);
     buildServerWithFallbackServerCredentials(
-        xdsClientWrapperForServerSds, InsecureServerCredentials.create(), downstreamTlsContext);
-
-    XdsClient.LdsResourceWatcher listenerWatcher = xdsClientWrapperForServerSds
-        .getListenerWatcher();
+            InsecureServerCredentials.create(), downstreamTlsContext);
 
     SimpleServiceGrpc.SimpleServiceBlockingStub blockingStub = newApi
         ? getBlockingStub(upstreamTlsContext, "foo.test.google.fr") :
         getBlockingStub(upstreamTlsContext, "foo.test.google.fr");
```

Comment:
Use `OVERRIDE_AUTHORITY` in lines 307 and 308?

Reply:
I found 4 occurrences that can be replaced with the constant. Lines 160, 187, 248 and 277. IDE should help find them all